lib/commons/dom/is-visible.js

```diff
@@ -112,50 +113,74 @@ function isVisible(el, screenReader, recursed) {
     );
   }
 
-  const vNode = getNodeFromTree(el);
+  const vNode = el instanceof AbstractVirtualNode ? el : getNodeFromTree(el);
```

Comment:
The general strategy is check things that virtual node has access to, followed by things only a true DOM node has access to. A detatched virtual node is visible.